.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,73 @@
+---
+name: "🐛 Bug Report"
+description: Report a bug
+title: "(short issue description)"
+labels: [bug, needs-triage]
+assignees: []
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: What is the problem? A clear and concise description of the bug.
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected Behavior
+      description: |
+        What did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: current
+    attributes:
+      label: Current Behavior
+      description: |
+        What actually happened?
+        
+        Please include full errors, uncaught exceptions, stack traces, and relevant logs.
+        If service responses are relevant, please include wire logs.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction Steps
+      description: |
+        Provide a self-contained, concise snippet of code that can be used to reproduce the issue.
+        For more complex issues provide a repo with the smallest sample that reproduces the bug.
+        
+        Avoid including business logic or unrelated code, it makes diagnosis more difficult.
+        The code sample should be an SSCCE. See http://sscce.org/ for details. In short, please provide a code sample that we can copy/paste, run and reproduce.
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Possible Solution
+      description: |
+        Suggest a fix/reason for the bug
+    validations:
+      required: false
+  - type: textarea
+    id: context
+    attributes:
+      label: Additional Information/Context
+      description: |
+        Anything else that might be relevant for troubleshooting this bug. Providing context helps us come up with a solution that is most useful in the real world.
+    validations:
+      required: false
+  - type: input
+    id: sdk-version
+    attributes:
+      label: Targeted .NET platform

```

Comment:
This doesn't have the package version or some of the other .NET-specific helptext we added to the other repos e.g. https://github.com/aws/aws-lambda-dotnet/blob/68e5159716d3f275087aa33e08dc73c8bf56e7f2/.github/ISSUE_TEMPLATE/bug-report.yml

Is that intentional?

Reply:
I think this was somewhat ported from deleted [.github/ISSUE_TEMPLATE/bug-report.md](https://github.com/aws/aws-extensions-for-dotnet-cli/pull/206/files#diff-5870a741e2fa8d0b03a88d7b036d408ee7e6954051fcc5d7978269bf6d1ff242). Please confirm if we would like to add other fields.

Reply:
Copy paste the same placeholder, same for environment

Reply:
Done in bug and feature request templates.

Reply:
I think for bugs it'd be good to collect the version of the CLI extension that a user is using (e.g. via ` dotnet tool list -g`)

Not sure "Targeted .NET platform" is as relevant here though as it is for other repos:
1. Are you looking for `dotnet --version`?
2. Or, are we really after the target platform of whatever we're trying to deploy? If so, is that covered by reproduction steps

Reply:
I think `dotnet tool list -g` makes more sense for bugs. May be using text area would also help. Should we remove this field from feature request all together?

Reply:
include both, tool install and runtime that tool is using is important to debug.

Reply:
Done. Please review.